src/content/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability.mdx

```diff
@@ -41,11 +46,9 @@ Our platform goes beyond simple monitoring by offering you observability: data i
   A comparison between simple monitoring and observability.
 </figcaption>
 
-## Get your data into New Relic [#integrations]
+## Get your data into New Relic with our quickstarts [#quickstarts]
```

Comment:
I thought we weren't supposed to rename anchors in case a doc somewhere else already links to the original name?

Reply:
Oh, I missed that one. Thanks!